ocs_ci/helpers/managed_services.py

```diff
@@ -768,10 +768,12 @@ def verify_faas_provider_storagecluster(sc_data):
         sc_data (dict): storagecluster data dictionary
 
     """
-    log.info(
-        f"allowRemoteStorageConsumers: {sc_data['spec']['allowRemoteStorageConsumers']}"
-    )
-    assert sc_data["spec"]["allowRemoteStorageConsumers"]
+    if version.get_semantic_ocs_version_from_config() < version.VERSION_4_19:
+        log.info(
+            f"allowRemoteStorageConsumers: {sc_data['spec']['allowRemoteStorageConsumers']}"
+        )
```

Comment:
Is the info level information about this value necessary? What about to put the message directly to assert statement below like 

```python
assert sc_data["spec"]["allowRemoteStorageConsumers"], (
    f"allowRemoteStorageConsumers: {sc_data['spec'].get('allowRemoteStorageConsumers') is not defined"
)
```

I tink that if value will not be defined in sc_data you will fail in logging with key error exception instead of in assert statement.

Reply:
I did not change the test, this line has existed like that. I only added excluding if condition

Reply:
ah, right. so you can keep it like that than. Didn't realized you didn't touched this part